ui/app/components/secret-engine/list.hbs

```diff
@@ -43,7 +43,17 @@
     <div>
       <div class="has-text-grey is-grid align-items-center linked-block-title">
         {{#if backend.icon}}
-          <Hds::TooltipButton @text={{or backend.engineType backend.path}} aria-label="Type of backend">
```

Comment:
More than moving files, I amended the tooltip helper here and added the version.